src/test/scala/edu/berkeley/cs/rise/opaque/OpaqueOperatorTests.scala

```diff
@@ -411,6 +411,12 @@ trait OpaqueOperatorTests extends FunSuite with BeforeAndAfterAll { self =>
     df.filter($"word".contains(lit("1"))).collect
   }
 
+  testAgainstSpark("between") { securityLevel =>
+    val data = for (i <- 0 until 256) yield(i.toString, i)
+    val df = makeDF(data, securityLevel, "word", "count")
+    df.filter($"count".between(3, 5)).collect
```

Comment:
Can you increase the range of the between? Maybe something like `between(50, 150)`

Reply:
Changed